linode/manager
# refactor: [M3-8813] - Remove use of Redux for viewing StackScript details

## Description 📝

- I'm still trying to remove use of Redux and recompose 🗑️. We're slowly getting there 🐌 
- This removes the Redux state for managing a StackScript details dialog 🗑️ 
  - This dialog is simple enough that we don't need Redux to manage it. We can just use local state
- Hopefully I'll be able to begin refactoring more StackScripts code soon. It is a big mess and is one of the last features that doesn't use React Query 🤞 


## Preview 📷
> [!note]
> There are no UI changes

![Screenshot 2024-10-30 at 3 57 55 PM](https://github.com/user-attachments/assets/ea1f7ed8-65c4-456a-a6e8-913186e958d7)


## How to test 🧪

- These changes should only affect the Linode Rebuild Dialog
- Verify you can still view a StackScript's detail in the Linode Rebuild Dialog

https://github.com/user-attachments/assets/42f253fa-ba56-46b3-b481-3c4189b3bd5b

## As an Author I have considered 🤔
